setup.cfg

```diff
@@ -19,17 +19,22 @@ install_requires =
 spacy_architectures =
     spacy-experimental.Bilinear.v1 = spacy_experimental.biaffine_parser.bilinear:build_bilinear
     spacy-experimental.PairwiseBilinear.v1 = spacy_experimental.biaffine_parser.pairwise_bilinear:build_pairwise_bilinear
+    experimental.span_finder_model.v1 = spacy_experimental.span_finder.span_finder_model:build_finder_model
 spacy_factories =
     experimental_arc_labeler = spacy_experimental.biaffine_parser.arc_labeler:make_arc_labeler
     experimental_arc_predicter = spacy_experimental.biaffine_parser.arc_predicter:make_arc_predicter
     experimental_edit_tree_lemmatizer = spacy_experimental.edit_tree_lemmatizer.edit_tree_lemmatizer:make_edit_tree_lemmatizer
     experimental_char_tagger_tokenizer = spacy_experimental.char_tokenizer.char_tagger_tokenizer:make_char_tagger_tokenizer
     experimental_char_ner_tokenizer = spacy_experimental.char_tokenizer.char_ner_tokenizer:make_char_ner_tokenizer
+    experimental_span_finder = spacy_experimental.span_finder.span_finder_component:make_span_finder
 spacy_scorers =
     spacy-experimental.tokenizer_scorer.v1 = spacy_experimental.char_tokenizer.scorers:make_tokenizer_scorer
     spacy-experimental.tokenizer_senter_scorer.v1 = spacy_experimental.char_tokenizer.scorers:make_tokenizer_senter_scorer
+    experimental.span_finder_scorer.v1 = spacy_experimental.span_finder.span_finder_component:make_span_finder_scorer
 spacy_tokenizers =
     spacy-experimental.char_pretokenizer.v1 = spacy_experimental.char_tokenizer.char_pretokenizer:char_pretokenizer_v1
+spacy_misc =
+    experimental.span_finder_suggester.v1 = spacy_experimental.span_finder.span_finder_suggester:build_span_finder_suggester
```

Comment:
For consistency, I'd prefix all of these new ones with `spacy-experimental` like we've done for the current factories in this package.